xslt/geonetwork_update_backlogs_324/transform-catalog-mcp-1.4.xsl

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mcp="http://bluenet3.antcrc.utas.edu.au/mcp"
+                xmlns:gco="http://www.isotc211.org/2005/gco"
+                xmlns:gmd="http://www.isotc211.org/2005/gmd"
+                xmlns:gmx="http://www.isotc211.org/2005/gmx"
+                xmlns:geonet="http://www.fao.org/geonetwork"
+
+                exclude-result-prefixes="xsl mcp gco gmd gmx geonet"
+>
+
+    <xsl:import href="xslt/geonetwork_update_backlogs_324/ciResponsibleParty.xsl"/>
+
+    <xsl:template match="node()|@*">
+        <xsl:copy>
+            <xsl:apply-templates select="node()|@*"/>
+        </xsl:copy>
+    </xsl:template>
+
+    <xsl:template match="mcp:MD_Metadata/gmd:contact">
```

Comment:
Make that with 'Derwent Estuary Program' in the title.  Derwent Estuary gives more than 65 results.